### PR TITLE
Updates to documentation for icons/images

### PR DESCRIPTION
**Pull request by ashenden**
We need to add some additional documentation for our expectations for package icons and images.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -61,6 +61,14 @@ The required fields are:
 - maintainer
 - description
 
+While `images` is an optional field, it is highly recommended you include icons and screenshots in your package and update the path definitions accordingly.  Specifications are as follows:
+* `icon-small`: 48px (w) x 48px (h)
+* `icon-medium`: 96px (w) x 96px (h)
+* `icon-large`: 256px (w) x 256px (h)
+* `screenshots[...]`: 1200x675
```
Comment (jsancio) [inline README.md]: Should this match the format above? E.g. `1200px (w) x 675px (h)`

**Comment by jsancio** [comment]
Looks good to me :birthday: . One minor comment.

**Comment by ConnorDoyle** [comment]
:+1: thanks John

File: README.md (hunk 1 above)
Comment (ashenden) [inline README.md]: @jsancio yes.  Can you make the change?

Comment (ConnorDoyle) [inline README.md]: FTFY

**Pull request merged**